demisto_sdk/commands/test_content/IntegrationsLock.py

```diff
@@ -10,9 +10,10 @@
 
 LOCKS_PATH = 'content-locks'
 BUCKET_NAME = os.environ.get('GCS_ARTIFACTS_BUCKET')
-CIRCLE_BUILD_NUM = os.environ.get('CIRCLE_BUILD_NUM')
-WORKFLOW_ID = os.environ.get('CIRCLE_WORKFLOW_ID')
+BUILD_NUM = os.environ.get('CI_BUILD_ID')
+WORKFLOW_ID = os.environ.get('CI_PIPELINE_ID')
 CIRCLE_STATUS_TOKEN = os.environ.get('CIRCLECI_STATUS_TOKEN')
+GITLAB_STATUS_TOKEN = os.environ.get('GITLAB_STATUS_TOKEN')
```

Comment:
I just want to confirm I understand correctly. What's the reason we aren't unifying this to a single `STATUS_TOKEN` environment variable (or something along those lines)? Because for the span of time in which we're going to have the build running in both CircleCI and GitLab, we'll be considering both builds with regards to locking/unlocking integrations for a test playbook?